demo/run-osm-demo.sh

```diff
@@ -138,6 +142,13 @@ fi
 
 ./demo/configure-app-namespaces.sh
 
+if [[ "$USE_PRIVATE_REGISTRY" = true && "$DEPLOY_ON_OPENSHIFT" = true ]]; then
+    oc secrets link bookbuyer "$CTR_REGISTRY_CREDS_NAME" --for=pull -n "$BOOKBUYER_NAMESPACE"
```

Comment:
I don't think this will work because the service accounts aren't created yet. You should instead wrap the existing calls to `oc secrets link` since those are in the correct place (called after serviceaccount creation)

Reply:
I see, I didn't realize it was already in the deploy-bookbuyer / store scripts